packages/components/inputs/styles.js

```diff
@@ -29,20 +28,26 @@ const getInputStyles = (props, theme) => {
     padding: 0 ${overwrittenVars.spacingS};
     transition: border-color ${overwrittenVars.transitionStandard},
       background-color ${overwrittenVars.transitionStandard},
-      color ${overwrittenVars.transitionStandard};
+      color ${overwrittenVars.transitionStandard},
+      box-shadow ${overwrittenVars.transitionStandard};
     width: 100%;
 
     &::placeholder {
       color: ${overwrittenVars[designTokens.placeholderFontColorForInput]};
     }
     &:active,
-    &:focus {
+    &:focus,
+    &:hover {
```

Comment:
This is basically the main change regarding the hover style: it now has the style that used to be applied only on focus/active.